action.yml

```diff
@@ -29,18 +33,28 @@ runs:
       with:
         result-encoding: json
         script: |
+          const fs = require("fs");
+          const path = require("path");
+
+          const gap = process.env.GITHUB_ACTION_PATH;
           const cliPath = '/node_modules/@replayio/replay';
-          const cli = require(process.env.GITHUB_ACTION_PATH + cliPath);
-          const { source } = require(process.env.GITHUB_ACTION_PATH + cliPath + "/metadata");
-          const upload = require(process.env.GITHUB_ACTION_PATH + '/dist/upload.js');
-          const buildSourceMetadata = require(process.env.GITHUB_ACTION_PATH + '/dist/source.js');
+          const cli = require(gap + cliPath);
+          const { source } = require(gap + cliPath + "/metadata");
+          const upload = require(gap + '/dist/upload.js');
+          const buildSourceMetadata = require(gap + '/dist/source.js');
           const ids = await upload({
             cli,
             apiKey: '${{ inputs.apiKey || null }}',
             public: ${{ inputs.public || false }},
             filter: '${{ inputs.filter || null }}',
             metadata: await buildSourceMetadata(source, context, github)
           });
-          return ids;
+
+          // Write output to a file
+          const output = path.join(gap, "recordings.json");
+          fs.writeFileSync(output, JSON.stringify(ids));
+
+          core.setOutput('recordings', ids);
```

Comment:
does core need to be required? Or does it just come with GHA? https://github.com/actions/toolkit/tree/master/packages/core#import-the-package

Reply:
It comes with `actions/github-script`